en/docs/guides/login/validate-jwt-using-jwks.md

```diff
@@ -0,0 +1,63 @@
+# Validate a JWT Based on JWKS
```

Comment:
This content is duplicated in guides/access-delegation/validate-jwt-based-on-jwks.md 

Shall we remove the page in access-delegation folder?

Reply:
+1 will remove it from there